apps/remix-ide/src/app/tabs/locales/en/pluginManager.json

```diff
@@ -33,6 +33,7 @@
   "pluginManager.ok": "OK",
   "pluginManager.cancel": "Cancel",
   "pluginManager.maintainedByRemix": "Maintained by Remix",
+  "pluginManager.maintainedExternally": "Be aware that the plugin is not maintained by Remix",
```

Comment:
Should either be (I prefer the 1st option) :
"Not maintained by Remix"
or
"Be aware that this plugin is not maintained by Remix"